### ISSUE TITLE
Alt text support

### ISSUE THREAD

**Issue by treuks** [Chatterino/api#440]
It would be super nice if we could have alt text from the images show up in the embeds.
The most obvious example I can give are the XKCD comics, where the alt text is super important.

![image](https://user-images.githubusercontent.com/18615981/221122670-fbabc831-30ad-47e3-b069-9695f787a45a.png)

**Comment by pajlada** [comment]
The link you're hovering over is not an image, it's a link to https://xkcd.com/2740/
The way we extract the image is using the `og:image` tag
![image](https://user-images.githubusercontent.com/962989/221145873-9cc34586-2fdd-49a2-be29-869f5ebdb90f.png)

Same thing with the title.

We cannot in a feasible way get the alt text from that website without making this an API call.
